### ISSUE TITLE
Debian package

### ISSUE THREAD

**Issue by sephalon** [linux-automation/usbsdmux#20]
**This is work-in-progress**

**Branches `debian` and `upstream` must be kept separately, do not merge into `master`. See [git-buildpackage documentation](https://honk.sigxcpu.org/projects/git-buildpackage/manual-html/gbp.html).**

I started to work on a Debian package in [sephalon/usbsdmux, `debian` branch](https://github.com/sephalon/usbsdmux/tree/debian). After #18 and #19 have been resolved, this can be finished and brought upstream.

**Comment by sephalon** [comment]
I have droped this in favor of #37.
